src/_helpers.js

```diff
@@ -24,35 +24,7 @@ export const pickTensors = tree => {
   return objects;
 };
 
-export const torchToTF = (command, kwargs, logger) => {
-  const cmd_map = {
-    t: 'transpose',
-    __matmul__: 'matMul',
-    __truediv__: 'div',
-    __gt__: 'greater',
-    eq: 'equal',
-    'torch.argmax': ['argMax', [], [kwargs['dim']]],
-    float: ['cast', ['float32']],
-    'torch.nn.functional.relu': 'relu',
-    'torch.nn.functional.softmax': ['softmax', [], [kwargs['dim']]]
-  };
-
-  if (command in cmd_map) {
-    return cmd_map[command];
-  }
-
-  // In Python, some commands in TensorFlow and PyTorch are submitted with double-underscores to avoid method collision
-  // Since we don't need this nonsense in TensorFlow.js... let's strip all the underscores
-  command = command.split('_').join('');
-
-  // Sometimes PyTorch commands come with "torch." on the beginning, strip that out
-  command = command.split('torch.').join('');
-
-  // If the command as it's currently named exists in TensorFlow.js already, return the command name
-  if (Object.hasOwnProperty.call(tf, command)) return command;
-
-  // If not, we will need to do a lookup of the command in question
-  logger.log(CANNOT_FIND_COMMAND(command));
-
-  return null;
+export const torchToTF = command => {
```

Comment:
I'd just remove that helper function and placed
`const threepio = new Threepio('torch', 'tfjs', tf);`
on top of message.js, so we init it only once.